aws/config.go

```diff
@@ -165,6 +165,11 @@ type Config struct {
 	// in the ARN, when an ARN is provided as an argument to a bucket parameter.
 	S3UseARNRegion *bool
 
+	// Set this to `true` to have header names normalized in API responses.
+	// This settings affects S3 x-amz-meta headers, and normalizes the key names
+	// to lower case.
+	NormalizeHeaders *bool
```

Comment:
Might help discoverability to name this, LowerCaseHeaderMaps, and add code generated documentation to all member refs that target a shape with headers bindings to include documentation about this new flag, and how to enable it.